### DISCUSSION TITLE
No disk space on manager /

### DISCUSSION THREAD

**Discussion by wtl06639** [Security-Onion-Solutions/securityonion#4020] [Unsupported Versions]
The so-services on my manager (distributed deployment) are not able to start anymore because of no disk space. It appears the issue lies within  /var/lib/docker/volumes that keep getting created daily but never removed. I had over 100GB space for / but it has filled up. Should these older volumes be removed on their own? . Docker is taking up 95 GB and the volumes directory is taking up 80 GB. Is there a way to free up this space?

The manager is also running on a VM. I added more disk space but it does not appear to show on the system itself. Is there an easy way to have the see the new storage space and allocate it to /?  If not, if i were to reinstall the whole OS manager (same setting) would it cause problems with the heavy node sensors that are currently connected to it?

**Comment by cm-ops** [comment]
You could try stopping all the containers and running `docker system prune -a -f --volumes`. Then run a `salt-call state.hightstate`. 

As far as adding disk space, if you set up using LVM, you could use that to add the disk space.

**Comment by TOoSmOotH** [comment]
#4073 is in to start purging unused containers.
